dockerfiles/theia/src/setup.sh

```diff
@@ -2,6 +2,12 @@
 set -e
 set -u
 
+yum install -y sudo
```

Comment:
if possible, merge it with other yum install instruction post github-check

Reply:
To me it is clearer when we install components in its logical group/place.
Just curious, what is advantage of such merging?

Reply:
speed of build (like searching for mirrors again)